fem/linearform.hpp

```diff
@@ -14,18 +14,39 @@
 
 #include "../config/config.hpp"
 #include "lininteg.hpp"
+#include "linearform_ext.hpp"
 #include "gridfunc.hpp"
 
 namespace mfem
 {
 
+/** @brief Enumeration defining the assembly level for linear form classes
+ *  derived from Vector. */
+enum class LinearAssemblyLevel
+{
```

Comment:
I think it will be ideal to just reuse `AssemblyLevel` here (and interpret it appropriately for linear forms).

If that's not possible, I will at least suggest changing the name to `LinearFormAssemblyLevel` or maybe `LFAssemblyLevel`.

Reply:
We are using `AssemblyLevel` from `BilinearForm` now.